rasa_sdk/executor.py

```diff
@@ -188,7 +188,17 @@ def _import_submodules(
         :rtype: dict[str, types.ModuleType]
         """
         if isinstance(package, str):
-            package = importlib.import_module(package)
+            if package == "actions":
+                try:
+                    package = importlib.import_module(package)
+                except ImportError:
+                    logger.exception(f"Could not import actions. Actions not found")
+                    exit()
+            else:
+                package = importlib.import_module(package)
+                if type(package) == str:
```

Comment:
why would the imported module be a `str`?

Reply:
Yes, it makes more sense calling sys.exit(1) right after Import Error in register_package() function. Doing the same.

Reply:
@ricwo More changes?

Reply:
@gitpranjal yes, please don't forget to add a changelog entry describing the change